charts/consul/values.yaml

```diff
@@ -471,6 +471,14 @@ server:
     runAsUser: 100
     fsGroup: 1000
 
+  # The container securityContext for the server pods.  In addition to the [Pod
+  # secuirtyContext](#v-server-securitycontext) this can set the capabilities
+  # of processes running in the container and ensure the root file systems in
+  # the container is read-only.
+  # @type: map
+  # @recurse: false
+  containerSecurityContext: null
```

Comment:
I think we need `server` within here to match `client` and to future-proof ourselves.